lib/util/sdp/simulcast.js

```diff
@@ -169,15 +171,52 @@ function getSSRCRtxPairs(section) {
   return new Map(getMatches(section, rtxPairPattern).map(pair => pair.reverse()));
 }
 
+/**
+ * Create SSRC attribute tuples.
+ * @param {string} section
+ * @param {'planb' | 'unified'} sdpFormat
+ * @returns {Array<[SSRC, ?string, Track.ID]>}
```

Comment:
Can you clarify what the middle type is? I guess MediaStream ID? And you've said it's always null for the Unified Plan case? But that's not always true—there may be `a=msid` attributes that use a value other than `-`. So why make it null?